packages/create/src/recipes/sveltekit/template/ts/src/lib/server/auth.__auth.ts

```diff
@@ -0,0 +1,6 @@
+import { createClient } from "edgedb";
+
+export const client = createClient({
+  //Note: when developing locally you will need to set tls  security to insecure, because the dev server uses  self-signed certificates which will cause api calls with the fetch api to fail.
```

Comment:
```suggestion
  // Note: when developing locally you will need to set tls security to insecure, because the dev server uses self-signed certificates which will cause api calls with the fetch api to fail.
```